src/js/tools/drawing/Lighten.js

```diff
@@ -41,19 +41,24 @@
   /**
    * @Override
    */
-  ns.Lighten.prototype.applyToolAt = function(col, row, color, frame, overlay, event, mouseButton) {
+  ns.Lighten.prototype.applyToolAt = function(col, row, frame, overlay, event, mouseButton) {
```

Comment:
(not related to your change) We should remove this unused mouseButton argument. Up to you if you want to do it with this PR or separately.